pages/_app.tsx

```diff
@@ -54,11 +53,9 @@ const WithStateContainerProviders = ({ children }: IProps) => (
                                     <DvmContracts.Provider>
                                       <DvmState.Provider>
                                         <DevMining.Provider>
-                                          <PerpetualState.Provider>
```

Comment:
this has been replaced by ContractState introduced in the emp2.0 update

Reply:
yes i believe so. We launched 2.0 compatibility last week or early this week and afaik the 1.0 contracts did not break. You can see in `utils/getAbi`, theres ways to normalize the state of the contracts